### PR TITLE
Add prefixes to balance contract

### PR DESCRIPTION
**Pull request by roman-khimov**
And some tests as well.

### CODE REVIEW

File: contracts/balance/migration_test.go [hunk 1]
```diff
@@ -60,7 +62,25 @@ func testMigrationFromDump(t *testing.T, d *dump.Reader) {
 	// try to update the contract
 	if notaryDisabled && prevPendingVotes {
 		c.CheckUpdateFail(t, "pending vote detected")
-		return
+		t.Fail()
```
Comment (cthulhu-rider) [inline contracts/balance/migration_test.go:65]: have u reached this? the change worth extension to tests of other contracts (and separation from the currentone)

File: contracts/balance/migration_test.go [hunk 2]
```diff
@@ -60,7 +62,25 @@ func testMigrationFromDump(t *testing.T, d *dump.Reader) {
...
+	}
+
+	var accounts = []util.Uint160{}
```
Comment (cthulhu-rider) [inline contracts/balance/migration_test.go:68]: cosmetic
```suggestion
	var accounts []util.Uint160
```
same below

Comment (cthulhu-rider) [inline contracts/balance/migration_test.go:68]: i'd also assert that items with these keys are removed from the storage after update

File: contracts/balance/migration_test.go (hunk 1 above)
Comment (roman-khimov) [reply contracts/balance/migration_test.go:65]: No. We don't have it in the current data set and we won't ever have it for obvious reasons, but I wanted to ensure that test doesn't silently "succeed". Can be reverted.

**Review by cthulhu-rider — APPROVED** [review]

**Pull request merged**